src/scripts/validation.js

```diff
@@ -52,7 +52,7 @@ const disableButton = (buttonEl, config) => {
 };
```

Comment:
* You need to **`disable`** the submit button in the `card form` if the inputs are **empty**. It’s needed because I can click the button and add an empty card, but it should **not** happen. After the first submission, the button gets activated, and you need to **deactivate** it again. Even if I can't add an empty card, the button should look **disabled** (gray) anyway and have a `disabled` attribute.
* You should **disable** the `avatar` submit button after submission as well https://snipboard.io/ilSakD.jpg 


Please, **export** the `disableButton` function from `validation.js`, to be able to use it in `index.js` for disabling


```js
export const disableButton = (buttonEl, config) => {
  buttonEl.disabled = true;
  buttonEl.classList.add(config.inactiveButtonClass);
};
```